src/screens/explore/Explore.jsx

```diff
@@ -1,7 +1,73 @@
 /** @format */
 
 import React from "react";
+import "./explore.css";
+
+import { VideoCard, Button } from "../../components";
+import { useVideos } from "../../context";
+import { videoFilter } from "../../utils";
 
 export const Explore = () => {
-  return <div className='main-container'>Explore</div>;
+  const {
+    videoState: { videos, filters },
+    videoDispatch,
+  } = useVideos();
+
+  const buttonTitle = [
+    {
+      title: "All Videos",
+      param: "allvideos",
+    },
+    {
+      title: "Javascript",
+      param: "javascript",
+    },
+    {
+      title: "React",
+      param: "react",
+    },
+    {
+      title: "Html",
+      param: "html",
+    },
+    {
+      title: "Css",
+      param: "css",
+    },
+  ];
```

Comment:
This too can be moved out side the component

Reply:
will do it.